spec/requests/api/status_spec.rb

```diff
@@ -2,13 +2,30 @@
 
 describe "API::StatusController" do
   let(:user) { create(:user) }
-  let!(:project) { create(:project) }
+  let!(:project) { create(:project, name: 'rails1') }
+  let!(:project_django) { create(:project, name: 'Django', platform: 'Pypi') }
+
   describe "GET /api/check", type: :request do
-    it "renders successfully" do
+    it "renders successfully with one" do
+      post "/api/check", params: {api_key: user.api_key, projects: [{name: 'rails1', platform: 'rubygems'}] }
```

Comment:
If you switch to  `projects: [{name: project.name, platform: project.platform}]` (and similar in the rest of the tests) then this will be less brittle to changes in the factory in the future